### ISSUE TITLE
Cannot get the simplest "Hello World" to work.

### ISSUE THREAD

**Issue by reid-spencer** [playframework/twirl#52]
TL;DR - https://github.com/reid-spencer/im-going-twirly

I'm working on a project, without Play, that uses Twirl. For some reason I can't get sbt to find the Twirl templates and compile them. I've been after this for many hours, read the docs, read the source code, and still can't figure out why my project doesn't work. I will sincerely appreciate any help in either pointing out my error, fixing the bug in Twirl, or at least making the correct magic incantation clearer in the document. 

Basically, I have a "HelloWorld" that looks like this:

``` scala
package com.example

import _root_.com.example.txt.hello_world

object Hello {
  def main(args: Array[String]): Unit = {
    println(hello_world(args))
  }
}
```

And, as you might infer, there should be a Twirl template at com.example.hello_world.scala.txt. There is and it looks like this: 

``` twirl
@(args: Array[String])
Hello, world. There are @{args.length} arguments. 
```

That's it. That's all the source code except for the build project. 

When I compile the project, I get this:

```
> compile
[info] Updating {file:/Users/reidspencer/Code/im-going-twirly/}twirl-bug...
[info] Resolving org.fusesource.jansi#jansi;1.4 ...
[info] Done updating.
[info] Compiling 1 Scala source to /Users/reidspencer/Code/im-going-twirly/target/scala-2.10/classes...
[error] /Users/reidspencer/Code/im-going-twirly/src/main/scala/com/example/Hello.scala:3: object txt is not a member of package com.example
[error] import _root_.com.example.txt.hello_world
[error]                           ^
[error] /Users/reidspencer/Code/im-going-twirly/src/main/scala/com/example/Hello.scala:7: not found: value hello_world
[error]     println(hello_world(args))
[error]             ^
[error] two errors found
[error] (compile:compile) Compilation failed
[error] Total time: 1 s, completed Nov 16, 2014 8:20:49 PM
```

Here's my plugins.sbt:

``` scala
scalaVersion := "2.10.4"

scalacOptions ++= Seq("-deprecation", "-unchecked", "-feature", "-Xlint")

// To get more information during initialization
logLevel := Level.Debug

// The Typesafe repository
resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"

addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.0.3")
```

And, here's my Build.scala:

``` scala
import play.twirl.sbt.Import.TwirlKeys
import sbt.Keys._
import sbt._
import play.twirl.sbt.SbtTwirl

object Build extends Build {

  lazy val our_settings : Seq[Setting[_]] = Seq(
    name := "im-going-twirly",
    version := "1.0",
    scalaVersion := "2.11.4",
    libraryDependencies += "org.scalatest" %% "scalatest" % "2.1.6" % "test",
    libraryDependencies += "com.typesafe.play" %% "twirl-api" % "1.0.3",
    sourceDirectories in TwirlKeys.compileTemplates := (unmanagedSourceDirectories in Compile).value
  )

  lazy val project = Project("im-going-twirly", file("."))
    .enablePlugins(SbtTwirl)
    .settings(settings:_*)

  override def rootProject = Some(project)
}
```

I thought this would be all too simple to fail but it eludes me.

Thanks in advance for any time you spend on it.
